src/data/matrix/basic.lean

```diff
@@ -1442,6 +1442,11 @@ lemma conj_transpose_reindex [has_star α] (eₘ : m ≃ l) (eₙ : n ≃ o) (M
   (reindex eₘ eₙ M)ᴴ = (reindex eₙ eₘ Mᴴ) :=
 rfl
 
+lemma mul_transpose_eq_minor_mul_minor_transpose [fintype n] [fintype m] [semiring α]
+  (e : n ≃ m) (M : matrix n m α) :
+  M ⬝ Mᵀ = (M.minor id e) ⬝ (M.minor id e)ᵀ  :=
```

Comment:
I could see this as a `simp` lemma if it was defined the other way, or is there a reason that the original direction is preferred?
```suggestion
@[simp]
lemma mul_transpose_eq_minor_mul_minor_transpose [fintype n] [fintype m] [semiring α]
  (e : n ≃ m) (M : matrix n m α) :
  (M.minor id e) ⬝ (M.minor id e)ᵀ = M ⬝ Mᵀ :=
```

Reply:
Ah, it's the simp normal form linter that is not happy about this. I will have a closer look later.